Meziantou.Xunit.ParallelTestFramework/ParallelTestAssemblyRunner.cs

```diff
@@ -0,0 +1,15 @@
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Meziantou.Xunit;
+
+public class ParallelTestAssemblyRunner : XunitTestAssemblyRunner
```

Comment:
Does it need to be public? I think only `ParallelTestFramework` need to be public.

Reply:
We could make it internal, but I figured there was no harm in allowing others to extend the functionality in this package. The `XunitTestAssemblyRunner` class we're overriding is already public and we don't add any public/protected methods beyond what `XunitTestAssemblyRunner` already exposed.